docs/architecture/modern-web-apps-azure/develop-asp-net-core-mvc-apps.md

```diff
@@ -228,6 +294,16 @@ services.AddMvc(o => o.Conventions.Add(new FeatureConvention()));
 
 ASP.NET Core MVC also uses a convention to locate views. You can override it with a custom convention so that views will be located in your feature folders (using the feature name provided by the FeatureConvention, above). You can learn more about this approach and download a working sample from the MSDN Magazine article, [Feature Slices for ASP.NET Core MVC](https://docs.microsoft.com/archive/msdn-magazine/2016/september/asp-net-core-feature-slices-for-asp-net-core-mvc).
 
+### APIs and Blazor applications
```

Comment:
This needs a no-loc metadata.

Reply:
Not sure what that means.

Reply:
@ardalis, Sorry for being not so clear.

The no-loc metadata is used to block specific technical terms from being translated.

Blazor will get translated without the no loc metadata.

Reply:
@ardalis, see https://github.com/dotnet/docs/issues/19309 for more details.

Reply:
Ok so this just goes in the file header, like these?

https://github.com/dotnet/docs/pull/19337/files